web/react/utils/utils.jsx

```diff
@@ -303,18 +303,25 @@ var getYoutubeEmbed = function(link) {
     };
 
     var success = function(data) {
-        $('.video-uploader.'+youtubeId).html(data.data.uploader);
-        $('.video-title.'+youtubeId).find('a').html(data.data.title);
+        if(!data.items.length || !data.items[0].snippet) {
+          return;
+        }
+        metadata = data.items[0].snippet;
```

Comment:
This line could use a var at the start so that metadata doesn't hang around for longer than it needs to.